portfolio/src/main/java/com/google/sps/data/Task.java

```diff
@@ -0,0 +1,33 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+/** A person who comments on the page. */
+public final class Task {
+
+  private final long id;
+  private final String firstName;
+  private final String lastName;
+  private final String country;
+  private final String subject;
+
+  public Task(long id, String firstName, String lastName, String country, String subject) {
```

Comment:
Would "Comment" be a more descriptive name?

Also, consider using [AutoFactory](https://github.com/google/auto/tree/master/factory) to create this type of immutable data value class.